nengo/utils/compat.py

```diff
@@ -30,6 +33,24 @@ def ensure_bytes(s):
         assert isinstance(s, bytes)
         return s
 
+    if sys.platform.startswith('win'):
+
+        def replace(src, dst):
```

Comment:
Maybe add a comment here briefly describing why this is necessary. Might be worth referencing this PR, too, so it's easy to find the full explanation.